packages/edit-site/src/components/resizable-frame/index.js

```diff
@@ -154,13 +161,38 @@ function ResizableFrame( {
 		if ( remainingWidth > SNAP_TO_EDIT_CANVAS_MODE_THRESHOLD ) {
 			// Reset the initial aspect ratio if the frame is resized slightly
 			// above the sidebar but not far enough to trigger full screen.
-			setFrameSize( { width: '100%', height: '100%' } );
+			setFrameSize( INITIAL_FRAME_SIZE );
 		} else {
 			// Trigger full screen if the frame is resized far enough to the left.
 			setCanvasMode( 'edit' );
 		}
 	};
 
+	// Handle resize by arrow keys
+	const handleResizableHandleKeyDown = ( event ) => {
+		if ( ! [ 'ArrowLeft', 'ArrowRight' ].includes( event.key ) ) {
+			return;
+		}
+
```

Comment:
Please add `event.preventDefault();` here. This should help prevent screen readers from reading the button text description character by character when pressing the Right or Left arrow keys.
Edit: make sure to place it _after_ the check for arrow keys because other keys e.g. Tab still needs to work as expected.

Reply:
Done 👍 f5b7a8f